public/js/action-list.js

```diff
@@ -20,217 +20,216 @@
      */
     var parseSelectionQuery = function (queryString) {
         return queryString.split('|');
-    };
+    }
 
-    Icinga.Behaviors = Icinga.Behaviors || {};
+    class ActionList extends Icinga.EventListener {
+        constructor(icinga) {
+            super(icinga);
 
-    var ActionList = function (icinga) {
-        Icinga.EventListener.call(this, icinga);
+            this.on('click', '.action-list [data-action-item]:not(.page-separator), .action-list [data-action-item] a[href]', this.onClick, this);
+            this.on('close-column', this.onColumnClose, this);
 
-        this.on('click', '.action-list [data-action-item]:not(.page-separator), .action-list [data-action-item] a[href]', this.onClick, this);
-        this.on('close-column', this.onColumnClose, this);
+            this.on('rendered', '.container', this.onRendered, this);
+        }
 
-        this.on('rendered', '.container', this.onRendered, this);
-    };
+        onClick(event) {
+            var _this = event.data.self;
+            var $activeItems;
+            var $target = $(event.currentTarget);
+            var $item = $target.closest('[data-action-item]');
+            var $list = $item.closest('.action-list');
 
-    ActionList.prototype = new Icinga.EventListener();
+            if ($target.is('a') && !$target.is('.subject')) {
+                return true;
+            }
 
-    ActionList.prototype.onClick = function (event) {
-        var _this = event.data.self;
-        var $activeItems;
-        var $target = $(event.currentTarget);
-        var $item = $target.closest('[data-action-item]');
-        var $list = $item.closest('.action-list');
+            event.preventDefault();
+            event.stopImmediatePropagation();
+            event.stopPropagation();
 
-        if ($target.is('a') && (! $target.is('.subject') || event.ctrlKey || event.metaKey)) {
```

Comment:
In which way is changing the condition so fundamentally, related to class syntax? Please restore this.